src/main/java/com/puppycrawl/tools/checkstyle/XMLLogger.java

```diff
@@ -105,7 +105,7 @@ public XMLLogger(OutputStream outputStream, OutputStreamOptions outputStreamOpti
      * Sets the OutputStream.
      * @param outputStream the OutputStream to use
      **/
-    private void setOutputStream(OutputStream outputStream) {
+    public final void setOutputStream(OutputStream outputStream) {
```

Comment:
Please explain this change and why users need this method public?
@romani What is your thoughts?
Why isn't writer `final`?

Reply:
@rnveach I could make it package-private. Previously I have updated `outputStream` in tests via reflection, but @romani asks me to remove it

Reply:
I did not noticed this change, it might be recent due to my request to make two executions on the same Logger.
making it public is not ok.
@soon , we need to come back to test method that do execution on same logger and the same stream and verify that on second execution we have two lines with `error line=\"1\" column=\"1\" severity=\"error\"`